tools/CoherenceBuild/CoherenceBuild.cs

```diff
@@ -152,5 +155,66 @@ private static string FindLatest(string repoDirectory)
                 .FirstOrDefault();
         }
 
+        private void GenerateDependenciesFile()
+        {
+            var project = new XElement("Project");
+            var propertyGroup = new XElement("PropertyGroup");
+            project.Add(propertyGroup);
+
+            var universeCoherence = _reposToProcess.First(repo => string.Equals(repo.Name, "UniverseCoherence", StringComparison.OrdinalIgnoreCase));
```

Comment:
I see. Are you referring to this? https://github.com/aspnet/Coherence/blob/rel/2.0.0-preview2/tools/CoherenceBuild/CoherenceBuild.cs#L86